openssl/src/pkey.rs

```diff
@@ -627,6 +683,63 @@ impl PKey<Private> {
             .map(|p| PKey::from_ptr(p))
         }
     }
+
+    #[cfg(ossl111)]
+    fn private_key_from_raw_bytes(
```

Comment:
It seems like we can just make this public (taking in the `Id` type: https://docs.rs/openssl/0.10.35/openssl/pkey/struct.Id.html) and not need to have a bunch of different constructors.